src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -164,6 +165,10 @@ private async Task LoadComponentsBackgroundAsync(CancellationToken cancellationT
             // Load the telemetry service and tell it to start watching the solution for project info.
             var projectTelemetryService = _workspace.Services.GetRequiredService<IProjectTelemetryService>();
             projectTelemetryService.Start(this.DisposalToken);
+
+            // Load the todo comments service and tell it to start watching the solution for new comments
+            var todoCommentsService = _workspace.Services.GetRequiredService<ITodoCommentsService>();
+            todoCommentsService.Start(this.DisposalToken);
```

Comment:
All of these are VS-only services. I propose we remove IWorkspaceService and make them just regular MEF imports that can be retrieved from `ComponentModel.GetService<T>` (in a follow up PR).
I'd also remove the interfaces and rename the implementations to `VisualStudio(DesignerAttribute|ProjectTelemetry|TodoComments)Service`

Reply:
> I'd also remove the interfaces and just import 

I'm wary of that.  I really don't like the coupling there.  In general that's not great because now the entire surface area of that class is available to the consumer.  I really don't want to convey that (even if it's all private).  I really want to just say: you can grab this guy, and you can call "start" on it.

Reply:
I see, makes sense.

Reply:
How about renaming the interfaces to `IVisualStudioXxxService`?

Reply:
that soudns good to me :)